cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseTableTest.java

```diff
@@ -357,6 +368,80 @@ public void testTableWithPermissions() throws IOException {
     Assert.assertFalse(admin.exists());
   }
 
+  @Test
+  public void testScannerCache() throws Exception {
+    String tableName = "scanCache";
+    // note: it appears that HBase only enforces the scanner timeout after 10 seconds.
+    // setting it to 3 seconds does not mean it will actually fsail after 3 sweconds.
+    // therefore we have to cross the 10 seconds. here: 1200 times 10ms sleep.
+    int numRows = 1200;
+    DatasetAdmin admin = getTableAdmin(CONTEXT1, tableName);
```

Comment:
How did you create it? Inside this test? This test is weird in that it does not use the DatsetService and does not store the dataset properties. Inside this test, you need to provide the dataset properties every time you call getTable().